articles/resources/relnotes.md

```diff
@@ -17,6 +17,15 @@ For installation instructions, please refer to the [install guide](xref:microsof
 
 For update instructions, please refer to the [update guide](xref:microsoft.quantum.update).
 
+## Version X.XX.XXXXXXXX
+
+*Release date: November 23rd, 2020*
+
+- Improved compiler performance due to faster reference loading.
+- Updated Q# grammar for language repository. Fixed lexer bug with double literals and ranges. Updated grammar for namespaces, namespace elements, statements, and types and labels are used consistently. 
```

Comment:
I should clarify what the grammar deliverable is - it's not an update to any existing piece, but instead a new implementation of Q#'s syntax as an ANTLR grammar. I think the release notes should phrase it as a new part of the Q# specification:

```suggestion
- Added an [ANTLR grammar for Q#](https://github.com/microsoft/qsharp-language/tree/main/Specifications/Language/5_Grammar) to the Q# language specification.
```